### PR TITLE
[bridge 32/n] add BlocklistCommitteeAction and its encoding

### PR DESCRIPTION
**Pull request by longbowlu**
## Description 

as title

## Test Plan 

unit tests

---
If your changes are not user-facing and do not break anything, you can skip the following section. Otherwise, please briefly describe what has changed under the Release Notes section.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: crates/sui-bridge/src/error.rs [hunk 1]
```diff
@@ -41,6 +41,8 @@ pub enum BridgeError {
     InvalidBridgeAuthority(BridgeAuthorityPublicKeyBytes),
     // Authority's base_url is invalid
     InvalidAuthorityUrl(BridgeAuthorityPublicKeyBytes),
+    // Invalid Bridge client signing request
+    InvalidBridgeClientSigningRequest(String),
```
Comment (emmazzz) [inline crates/sui-bridge/src/error.rs:45]: This doesn't seem used in this pr?

File: crates/sui-bridge/src/types.rs [hunk 2]
```diff
@@ -478,6 +564,30 @@ mod tests {
 
     use super::*;
 
+    #[test]
+    fn test_bridge_authority_public_key_bytes_to_eth_address() {
```
Comment (emmazzz) [inline crates/sui-bridge/src/types.rs:568]: Should this test be in crypto.rs?

File: crates/sui-bridge/src/types.rs [hunk 3]
```diff
@@ -249,60 +365,13 @@ impl BridgeAction {
         bytes.extend_from_slice(BRIDGE_MESSAGE_PREFIX);
         match self {
             BridgeAction::SuiToEthBridgeAction(a) => {
```
Comment (emmazzz) [inline crates/sui-bridge/src/types.rs:367]: Here is it possible to do something like
```suggestion
            BridgeAction::SuiToEthBridgeAction(a) | BridgeAction::EthToSuiBridgeAction(a) |  BridgeAction::BlocklistCommitteeAction(a) => { 
                bytes.extend_from_slice(&a.to_bytes());
            }
```

**Review by emmazzz — APPROVED** [review]

File: crates/sui-bridge/src/types.rs (hunk 3 above)
Comment (longbowlu) [reply crates/sui-bridge/src/types.rs:367]: it does not, they need to be the same type.

**Pull request merged**